content/python/concepts/time-module/time-module.md

```diff
@@ -0,0 +1,19 @@
+---
+Title: 'Time Module'
+Description: 'Python has a built-in time module that we can use for assorted time functions. The following functions are provided by this module.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+  - 'Web Development'
+Tags:
+  - 'Methods'
+  - 'Constants'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/front-end-engineer-career-path'
+---
+
+Python has a built-in `time` module that we can use for time related tasks.
+
+The following functions are provided by this module.
```

Comment:
I think  at this line, we should include some brief information about terms, abbreviations, and phrases that are commonly used when working with the `time` module, such as:

* Epoch and how it is platform-dependent.
  * How seconds and leap-seconds are calculated according to the epoch.
* Timezones like UTC (formerly GMT).
* Certain events like daylights savings time (though I wonder how things might be effected if it's done away in the US if this [proposed bill](https://www.reuters.com/world/us/us-senate-approves-bill-that-would-make-daylight-savings-time-permanent-2023-2022-03-15/) is signed into law).

Maybe we should make a separate `struct_time` entry for a future PR.